src/pages/GithubApp/Components/Container.jsx

```diff
@@ -0,0 +1,228 @@
+            No results!
+          </div>
+        )}
+      </div>
+      <div className="dark:bg-gDarkBlue mt-5 bg-white rounded-xl shadow-2xl px-5 py-10 md:px-10 2xl:pl-40 2xl:relative transition duration-300">
+        <div className="flex items-center justify-around md:justify-start 2xl:ml-5 ">
+          <img
+            className="rounded-full w-[100px] 2xl:absolute 2xl:left-10 2xl:top-10 2xl:w-[130px] transition duration-300"
+            src={githubData.avatar_url}
+            alt=""
+          />
+          <div className="md:ml-8">
+            <h1 className="dark:text-gLightLightGray text-xl font-[900] 2xl:text-2xl transition duration-300">
+              {githubData.name}
+            </h1>
+            <p className="text-gLightBlue">{githubData.username}</p>
+            <p className="dark:text-gLightLightGray text-gLightGray 2xl:absolute right-10 top-10 transition duration-300">
+              Joined {githubData.join_date}
+            </p>
+          </div>
+        </div>
+        {githubData.bio === null ? (
+          <p className="dark:text-gLightLightGray text-gLightGray mt-8 leading-[25px] 2xl:ml-14 transition duration-300">
+            No bio yet
+          </p>
+        ) : (
+          <p className="dark:text-gLightLightGray text-gLightGray mt-8 leading-[25px] 2xl:ml-14 transition duration-300">
+            {githubData.bio}
+          </p>
+        )}
+        <div className="dark:bg-gDarkDarGray mt-14 bg-gLightLightGray px-5 py-7 rounded-xl flex justify-around 2xl:p-4  2xl:ml-12 transition duration-300">
+          <div className="text-center md:text-start">
+            <p className="dark:text-gLightLightGray text-gLightDarkBlue transition duration-300">
+              Repos
+            </p>
+            <p className="dark:text-gLightLightGray mt-2 font-[900] md:text-2xl transition duration-300">
+              {githubData.public_repos}
+            </p>
+          </div>
+          <div className=" text-center md:text-start ">
+            <p className="dark:text-gLightLightGray text-gLightDarkBlue transition duration-300">
+              Followers
+            </p>
+            <p className=" dark:text-gLightLightGray mt-2 font-[900]  md:text-2xl transition duration-300">
+              {githubData.followers}
+            </p>
+          </div>
+          <div className=" text-center md:text-start ">
+            <p className="dark:text-gLightLightGray text-gLightDarkBlue transition duration-300">
+              Following
+            </p>
+            <p className="dark:text-gLightLightGray mt-2 font-[900] md:text-2xl transition duration-300">
+              {githubData.following}
+            </p>
+          </div>
+        </div>
+        <div className=" w-[100%] flex flex-col justify-around flex-wrap mt-8 2xl:justify-start 2xl:ml-12 2xl:flex-row ">
+          <div>
+            <div className="flex">
+              <svg height="20" width="14" xmlns="http://www.w3.org/2000/svg">
```

Comment:
SVGs in separate files, they make this file completely unreadable outside of any IDE that can fold the tags.